### PR TITLE
Move genre/language tags guideline to rules in the RC

### PR DESCRIPTION
**Pull request by Hiviexd**
RC proposal thread: https://osu.ppy.sh/community/forums/topics/1621448?n=1

got permission from a NAT to apply this

### CODE REVIEW

**Comment by Walavouchey** [comment]
the same change can be done in the translations without needing to outdate them

**Review by Walavouchey — DISMISSED** [review]

File: wiki/Ranking_Criteria/zh.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+outdated_translation: true
+outdated_since: fa75d43c8d94994611338e131f2341c895559044
+---
 # 谱面上架标准 (RC)
 
 *若想查阅特定游戏模式的谱面上架标准，请参阅 [osu!](osu!)、[osu!taiko](osu!taiko)、[osu!catch](osu!catch)、[osu!mania](osu!mania)。*
```
Comment (YumeMuzi) [inline wiki/Ranking_Criteria/zh.md:7]:
```suggestion
*若想查阅特定[游戏模式](/wiki/Game_mode)的谱面上架标准，请参阅 [osu!](osu!)、[osu!taiko](osu!taiko)、[osu!catch](osu!catch)、[osu!mania](osu!mania)。*
```

**Comment by Walavouchey** [comment]
@YumeMuzi does the moved line also need changing in the translation? ("should" was changed to "must")

**Comment by Pisapou** [comment]
About the **fr.md**, to avoid having to pr an update for just this small change, can you modify what it says below:

- So change the position of genre/language tags guideline.
- "Must" and "should" have the same translation in french so there is no need to touch the wording.

**Comment by YumeMuzi** [comment]
> @YumeMuzi does the moved line also need changing in the translation? ("should" was changed to "must")

sure, line 155: change the character `应`  to `必须`

**Comment by Hiviexd** [comment]
hopefully i did it correctly

**Pull request merged**